compiler/rustc_metadata/src/creader.rs

```diff
@@ -799,7 +799,7 @@ impl<'a, 'tcx> CrateLoader<'a, 'tcx> {
         self.inject_dependency_if(cnum, "a panic runtime", &|data| data.needs_panic_runtime());
     }
 
-    fn inject_profiler_runtime(&mut self, krate: &ast::Crate) {
+    fn inject_profiler_runtime(&mut self, _krate: &ast::Crate) {
```

Comment:
Question: if `_krate` isn't used... maybe we should remove it?

Reply:
Yeah, in hindsight my reasons for leaving it as-is don't make much sense, so I'll change it.